docs/snapshotters/devmapper.md

```diff
@@ -24,15 +24,15 @@ version = 2
 The following configuration flags are supported:
 * `root_path` - a directory where the metadata will be available (if empty
   default location for `containerd` plugins will be used)
-* `pool_name` - a name to use for the devicemapper thin pool. Pool name
+* `pool_name` - a name to use for the Device Mapper thin-pool. Pool name
   should be the same as in `/dev/mapper/` directory
-* `base_image_size` - defines how much space to allocate when creating the base device
+* `base_image_size` - defines how much space to allocate when creating thin device snapshots from the base (pool) device
 * `async_remove` - flag to async remove device using snapshot GC's cleanup callback
 * `discard_blocks` - whether to discard blocks when removing a device. This is especially useful for returning disk space to the filesystem when using loopback devices.
 * `fs_type` - defines the file system to use for snapshot device mount. Valid values are `ext4` and `xfs`. Defaults to `ext4` if unspecified.
```

Comment:
Can you also add default values to optional parameters? Somehow we have that only for `fs_type`...

Reply:
It looks like fs_type is the only optional parameter with a default value in https://github.com/containerd/containerd/blob/7eae7f206ca04fe32ab05cab760bb39fb1fcf2c7/snapshots/devmapper/config.go

Reply:
I believe others follow Go's zero value semantics. So by default, all boolean values are false and all string values are "". But people who operate containerd don't have to be aware about Go semantics and these default values should be documented.

Reply:
Ah I see what you mean - sure can do